File: Common/src/main/java/datatypes/iotdevices/Procuration.kt
package datatypes.iotdevices

import java.math.BigInteger
import java.util.*

data class Procuration(val messageChainID: String = "", val deviceID: String = "", val recipientPublicKey: BigInteger = BigInteger("0"), val dateFrom: Date = Date(0), val dateTo: Date = Date(0))

File: WebApp/src/app/Procuration.ts
exportclassProcuration {
  messageChainID:string;
deviceID: string;
recipientPublicKey: BigInteger;
dateFrom: Date;
dateTo: Date;


constructor(messageChainID: string, deviceID: string, recipientPublicKey: BigInteger, dateFrom: Date, dateTo: Date) {
    this.messageChainID = messageChainID;
    this.deviceID = deviceID;
    this.recipientPublicKey = recipientPublicKey;
    this.dateFrom = dateFrom;
    this.dateTo = dateTo;
  }
}
